packages/theming/android-theme/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@fluentui-react-native/android-theme",
+  "version": "0.0.1",
```

Comment:
Nit: I think we start at 0.1.0 for most new packages

Reply:
Noted. Thanks for letting me know 😄 👍🏻